scripts/gulp/run.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const { spawn } = require('child_process');
+
+/**
+ * Run a command and return a promise for when it completes.
+ *
+ * Output and environment is forwarded as if running a CLI command in the terminal
+ * or make.
+ *
+ * This function is useful for running CLI tools as part of a gulp command.
+ *
+ * @param {string} cmd - Command to run
+ * @param {string[]} args - Command arguments
+ * @param {object} options - Options to forward to `spawn`
+ * @return {Promise<string>}
+ */
+function run(cmd, args, options = {}) {
```

Comment:
I resurrected run() from the client (its since been removed since the frontend-shared/ folder was deleted), I modified it to be able to return stdout and stderr.